### PR TITLE
Implement Transaction processing card

### LINKED ISSUE #596: [TASK] Implement reusable TransactionProgressCard component
## Task Description

The purpose of this task is to implement a reusable `TransactionProgressCard ` component that can be exported for external use. 

## Task Deliverable

- [x]  Create a folder `TransactionProgressCard `
- [x]  Implement a `TransactionProgressCard ` component with the below mentioned props
- [x]  Ensure the implementation reflects the intended design
- [x]  Document each component with an example
- [x]  Ensure component is exportable
- [x] Add component documentation to `/stories` dir

### TransactionProgressCard Component

The are variant types of `TransactionProgressCard ` that are dictated by the props passed in. Below is a list of the components properties that are required. Please refer to the `TransactionProgressCard ` design system link at the bottom to view each different `TransactionProgressCard ` type that are made up in this component. 

**Component properties:**

- Type="in-progress | warning | completed ";
- State="default | collapsed | expanded";
- Darkmode="off | on";

**Example usage:**

```bash
<TransactionProgressCard
  type="completed"
  dark-mode="off"
  state="default"
/>
```

## Relevant links

- [Figma Design System](https://www.figma.com/file/H43lYnTCOAPjO9kTrrepe1/%F0%9F%95%B8%EF%B8%8F-Design-System?node-id=1200%3A22610)

### PR DESCRIPTION
**Pull request by AhmedKorim**
## Summary of changes
- Adding the reusable transaction processing card

### Reference issue to close (if applicable)
- Closes  #596 
### Screenshots

-----
### Code Checklist 

- [x] Tested
- [x] Documented

### CODE REVIEW

File: apps/stats-dapp/src/pages/Proposals.tsx [hunk 1]
```diff
@@ -195,7 +195,8 @@ const Proposals = () => {
       <Card>
         <TitleWithInfo
           title="Proposals Status"
-          variant="h5"
+          vbabel-virtual-resolve-base.js
```
Comment (nepoche) [inline apps/stats-dapp/src/pages/Proposals.tsx:198]: What is this?

File: apps/stats-dapp/src/pages/TransactionProgressCardShowCase.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import EthLogo from '@webb-tools/logos/chains/EthLogo';
+import {
+  TransactionPayload,
+  TransactionQueueCard,
+} from '@webb-tools/webb-ui-components/containers/TransactionProgressCard';
```
Comment (dutterbutter) [inline apps/stats-dapp/src/pages/TransactionProgressCardShowCase.tsx:6]: We no longer need to make use of showcase, and should remove all showcase related components. Please remove this file and if you could remove the other unnecessary showcase components as well. Instead make use of storybook stories to display new component as you did.

File: apps/bridge-dapp/src/types/index.ts [hunk 3]
```diff
@@ -53,3 +52,29 @@ export interface SocialConfigsType extends Omit<ExternalLink, 'name'> {
...
+ * The Webb color type
+ */
+export interface WebbColorsType {
+  mono: Record<ColorIndex, string>;
+  purple: Record<ColorIndex, string>;
+  blue: Record<ColorIndex, string>;
+  green: Record<ColorIndex, string>;
+  yellow: Record<ColorIndex, string>;
+  red: Record<ColorIndex, string>;
+}
```
Comment (dutterbutter) [inline apps/bridge-dapp/src/types/index.ts:80]: Do we need this? I thought the colour schema was being served from tailwind presets? @AtelyPham can you confirm?

**Review by dutterbutter — CHANGES_REQUESTED** [review]

File: apps/stats-dapp/src/pages/Proposals.tsx (hunk 1 above)
Comment (AhmedKorim) [reply apps/stats-dapp/src/pages/Proposals.tsx]: Removed typo

File: apps/stats-dapp/src/pages/TransactionProgressCardShowCase.tsx (hunk 2 above)
Comment (AhmedKorim) [reply apps/stats-dapp/src/pages/TransactionProgressCardShowCase.tsx:6]: Sure that makes sense, removing it with he others

File: apps/bridge-dapp/src/types/index.ts (hunk 3 above)
Comment (AhmedKorim) [reply apps/bridge-dapp/src/types/index.ts:80]: This was getting it working ,and this can be removed once @AtelyPham  confirms

**Comment by dutterbutter** [comment]
@AhmedKorim please test linting and building locally before pushing

File: apps/bridge-dapp/src/types/index.ts (hunk 3 above)
Comment (AhmedKorim) [reply apps/bridge-dapp/src/types/index.ts]: Removed

File: libs/webb-ui-components/src/stories/molecules/Disclaimer.stories.jsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { Disclaimer } from '../../components';
+
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+export default {
+  title: 'Design System/Molecules/CopyWithTooltip',
```
Comment (dutterbutter) [inline libs/webb-ui-components/src/stories/molecules/Disclaimer.stories.jsx:7]:
```suggestion
  title: 'Design System/Molecules/Disclaimer',
```

File: libs/webb-ui-components/src/stories/molecules/Disclaimer.stories.jsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { Disclaimer } from '../../components';
```
Comment (dutterbutter) [inline libs/webb-ui-components/src/stories/molecules/Disclaimer.stories.jsx:3]: Can we update the paths to follow other components:

```
@webb-tools/webb-ui-components/components
```

File: libs/webb-ui-components/src/stories/organisms/TransactionProccessingCard.stories.jsx [hunk 6]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { TransactionProgressCard } from '../../containers/TransactionProgressCard';
```
Comment (dutterbutter) [inline libs/webb-ui-components/src/stories/organisms/TransactionProccessingCard.stories.jsx:2]: Can we update the paths to follow other components:

```
@webb-tools/webb-ui-components/containers
```

File: libs/webb-ui-components/src/stories/organisms/TransactionQueueCard.stories.jsx [hunk 7]
```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+import { TransactionQueueCard } from '../../containers/TransactionProgressCard';
```
Comment (dutterbutter) [inline libs/webb-ui-components/src/stories/organisms/TransactionQueueCard.stories.jsx:2]: Can we update the paths to follow other components:

```
@webb-tools/webb-ui-components/containers 
```

**Review by dutterbutter — CHANGES_REQUESTED** [review]

**Pull request merged**